test/k8sT/manifests/ccnp-host-ingress-from-cidr-to-ports.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: "cilium.io/v2"
+kind: CiliumClusterwideNetworkPolicy
+description: "Host policy to allow traffic external to cluster inwards via CIDR and port"
+metadata:
+  name: "host-ingress-from-cidr-to-ports"
+spec:
+  nodeSelector: {}
+  ingress:
+  - toPorts:
+    - ports:
+      - port: "80"
+        protocol: TCP
+    fromCIDR:
+      - 192.168.36.13/32
```

Comment:
This won't work on non-vagrant CI builds, e.g. GKE, because the IP addrs will be different.

Reply:
All `fromCIDR`+`toPorts` tests do not currently run on GKE because they require a third node without Cilium. I don't know how easily we could enable that on GKE, but maybe something to look into as a follow up.